nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1265,6 +1275,28 @@ async fn ip_pool_update(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Fetch an IP pool
```

Comment:
I might summarize this as an IP Pool for Oxide services, specifically.

Reply:
Done: 726b4fd54fcd38568b1359e05ccc0dd84dd3f53c